AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/asset_processor_batch_tests.py

```diff
@@ -665,19 +661,19 @@ def test_validateDirectPreloadDependency_Found(self, asset_processor, ap_setup_f
 
         success, output = asset_processor.batch_process(capture_output=True, expect_failure=False)
         log = APOutputParser(output)
-        for _ in log.get_lines(-1, ["Preload circular dependency detected", "testc.prefab"]):
+        for _ in log.get_lines(-1, ["Preload circular dependency detected", "testb.auto_test_asset"]):
             error_line_found = True
 
+        if not error_line_found:
+            for line in output:
+                print(line)
+
```

Comment:
is this debug code?

Reply:
Yes, but I want to leave it in because other tests tend to print this out and it helps diagnose why the test failed.  Otherwise you just get a message saying "hey we didn't find the error we were looking for" and the only way to figure out what happened is to put this back in or tediously extract the run command and run AP from the command line yourself to see what errors occurred.